util/latlng_recon_service.py

```diff
@@ -16,13 +16,34 @@
 See latlng_recon_service_test.py for usage example.
 """
 
+import concurrent.futures
+import logging
 import requests
+from typing import Callable, Dict, List, NewType, TypeVar, Tuple
 
-_RECON_ROOT = 'https://api.datacommons.org/v1/recon/resolve/coordinate'
+LatLng = NewType('LatLng', Tuple[float, float])
+DCID = TypeVar('DCID')
+ResolvedLatLng = NewType('ResolvedLatLng', Dict[DCID, List[str]])
+
+_RECON_ROOT = "https://api.datacommons.org/v1/recon/resolve/coordinate"
 _RECON_COORD_BATCH_SIZE = 50
 
+logger = logging.getLogger(__file__)
+
+
+def _session(retries: int = 5, backoff_factor: int = 0.5) -> 'requests.Session':
+    """Returns a request session with retry."""
```

Comment:
please add judicious comments to shared libraries.

Reply:
Done in https://github.com/datacommonsorg/data/pull/779